### DISCUSSION TITLE
Keep cursor position after hsplit/vsplit

### DISCUSSION THREAD

**Discussion by rcorre** [helix-editor/helix#6175] [Q&A] [answered]
When I run `hsplit/vsplit`, helix moves the cursor to the first line in the new split.
Is there any way to have newly opened splits maintain the cursor position of the original split?
Sometimes I open a split so I can `goto_definition` on the word under the cursor, but I lose the position when I split.

**Comment by the-mikedavis** [comment, marked as answer]
You can use `<C-w>v` (`vsplit` command) for vertical splits and `<C-w>s` (`hsplit` command) for regular splits and they will copy the cursor location to the newly split buffer

**Reply by rcorre** [reply]
Thanks! Is it intentional that `vsplit` and `:vsplit` behave differently?

**Reply by the-mikedavis** [reply]
I'm not sure about the history of it but I believe that `:vsplit` is meant to act like `:open` and splitting the current buffer is a sort of fallback behavior when a filename is not also provided (like `:vsplit path/to/my/other/file.txt`). It would probably be fine to copy the selection in `:vsplit`/`:hsplit` though IMO
